package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "scripts": {
     "build": "tsc",
-    "watch": "tsc -w"
+    "pack": "webpack ./dist/src/index.js",
+    "start": "npm run build && npm run pack",
+    "test": "npm run start && playwright test"
```

Comment:
Why are you starting the server in a separate command?
we talked about opening the server programmatically in your javascript

Reply:
the server open in function.
the start command is for build and pack (line 7)

Reply:
is start is "saveed" command?